odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1986,6 +1986,12 @@ def distribute_branding(self, e, branding=None, parent_xpath='',
 
         node_path = e.get('data-oe-xpath')
         if node_path is None:
+            # Handle special case for jump points defined by the magic template <t>$0</t>
+            # No branding is allowed in this case since it points to a generic template.
+            no_branding = e.get('data-oe-no-branding')
+            if no_branding:
+                e.attrib.pop('data-oe-no-branding')
+                return
```

Comment:
Since you used a generic name like "data-oe-no-branding", it does not need explanation. But your comment here would be a better fit at the location where you used that data-oe-no-branding.